bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py

```diff
@@ -46,11 +46,21 @@ def __init__(self, module):
 
     def _fallback_getattr(self, name):
         # Try first in the global namespace, then in the ROOT namespace
-        # This allows to lookup e.g. ROOT.ROOT.Math as ROOT.Math
+        # and finally in gROOT (ROOT lists such as list of files,
```

Comment:
Minor thing, but maybe a list of three bullets with the three things we try would be more clear.